src/domain/community/community/community.service.authorization.ts

```diff
@@ -112,4 +120,41 @@ export class CommunityAuthorizationService {
 
     return updatedAuthorization;
   }
+
+  private appendPrivilegeRules(
+    authorization: IAuthorizationPolicy
+  ): IAuthorizationPolicy {
+    const privilegeRules: AuthorizationPolicyRulePrivilege[] = [];
+
+    const communityJoinPrivilege = new AuthorizationPolicyRulePrivilege(
+      [AuthorizationPrivilege.COMMUNITY_JOIN],
+      AuthorizationPrivilege.GRANT
+    );
+    privilegeRules.push(communityJoinPrivilege);
+
+    return this.authorizationPolicyService.appendPrivilegeAuthorizationRules(
+      authorization,
+      privilegeRules
+    );
+  }
+
+  private appendVerifiedCredentialRules(
+    authorization: IAuthorizationPolicy
+  ): IAuthorizationPolicy {
+    const verifiedCredentialRules: AuthorizationPolicyRuleVerifiedCredential[] =
+      [];
+
+    // todo: to be updated
```

Comment:
When will this be updated?

Reply:
As soon as I can get this together with the essif lab branching running together - should be an easy enough extension but do need them together.